Tests/RosalindTests/RosalindAcceptanceTests.swift

```diff
@@ -0,0 +1,61 @@
+import Command
+import FileSystem
+import Foundation
+import Path
+import Rosalind
+import SnapshotTesting
+import Testing
+
+@Suite(.serialized) struct RosalindAcceptanceTests {
+    private let commandRunner = CommandRunner()
+    private let fileSystem = FileSystem()
+    private let subject = Rosalind()
+
+    #if os(macOS)
+        @Test func apple_ios_app() async throws {
+            try await withFixtureInTemporaryDirectory("apple/ios_app") { temporaryDirectory, fixtureDirectory in
+                // Given
+                let xcodeprojPath = fixtureDirectory.appending(component: "App.xcodeproj")
+                let derivedDataDirectory = temporaryDirectory.appending(component: "derived-data")
+                try await commandRunner.run(
```

Comment:
I decided to compile on the tests to avoid including binaries in the repository. Since projects are small, I don't expect them to take a long time to compile.